ngi_pipeline/engines/piper_ngi/database.py

```diff
@@ -63,6 +63,8 @@ def create_database_populate_schema(location):
     if not os.path.exists( os.path.dirname(location) ):
         try:
             os.makedirs( os.path.dirname(location)) 
+        except OperationalError:
```

Comment:
Yea you're probably right. 
OperationalError relates to another problem you can get from ngi-pipeline by trying to access a db you don't own.
Updated.